pkg/webservice/handlers_test.go

```diff
@@ -386,8 +388,7 @@ func TestApplicationHistory(t *testing.T) {
 	imHistory = history.NewInternalMetricsHistory(5)
 	resp = &MockResponseWriter{}
 	getApplicationHistory(resp, req)
-	var appHist []dao.ApplicationHistoryDAOInfo
-	err = json.Unmarshal(resp.outputBytes, &appHist)
```

Comment:
this is required test since we should make sure `appHist` is not nil. 

`assert.Equal(t, len(appHist), 0, "empty response must have no records")` this test pass even though `appHist` is nil, and so we should add new `assert` for nil case

Reply:
done. check contHist too